src/util/Enums.js

```diff
@@ -12,6 +12,8 @@
 
 define({
 
+  WIDGET_CONTAINER_ID: 'okta-sign-in',
```

Comment:
How does this relate to `options.el`? The Widget may not always be rendered in an element with id `#okta-sign-in`.

Reply:
`#okta-sign-in` is not external to the widget, it's the top level element of the widget itself. `WIDGET_CONTAINER_ID` name might be confusing tho.

Reply:
It's exactly what Maunel said.
![screen shot 2019-01-02 at 9 11 19 am](https://user-images.githubusercontent.com/13139863/50603192-8b6eda00-0e6e-11e9-98ac-2c15d94945d1.png)